scripts/update-dockerhub-description.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+image_name="all-in-one"
+readme_path="./README.md"
```

Comment:
This is the root README? I think it's better to first check if there is a more narrow `./cmd/{binary}/README` file